spec.bs

```diff
@@ -452,6 +457,93 @@ To <dfn>build an interest group permissions url</dfn> given a [=origin=] |ownerO
 
 </div>
 
+<h3 id="interest-group-storage-maintenance">Interest Group Storage Maintenance</h3>
+
+There is a job that periodically [=performs storage maintenance=] on the [=user agent=]'s
+[=interest group set=]. It performs operations such as [=list/removing=] expired or excess
+[=interest groups=]. An [=interest group set=] must have no more than:
+  * 2000 [=interest group/owners=];
+  * 1000 [=regular interest groups=] per [=interest group/owner=];
+  * 20000 [=negative interest groups=] per [=interest group/owner=];
+  * <code>10\*1024\*1024</code> total [=interest group/estimated size=] of [=interest groups=] per
+    [=interest group/owner=].

```

Comment:
", including both [=regular interest groups=]s and [=negative interest groups=]s."?